src/main/java/com/mycompany/myapp/domain/Pool.java

```diff
@@ -0,0 +1,86 @@
+package com.mycompany.myapp.domain;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "pool")
+public class Pool {
+
+    @Id
+    @GeneratedValue(generator = "uuid")
+    @GenericGenerator(name = "uuid", strategy = "uuid2")
```

Comment:
Why it can't be:
```
@GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "sequenceGenerator")
@SequenceGenerator(name = "sequenceGenerator")
```

If we want to have 1 sequence on DB lets keep that

Reply:
because it is string?

Reply:
Makes sense XD